packages/atomic/src/components/search/search-box-suggestions/atomic-search-box-query-suggestions/atomic-search-box-query-suggestions.tsx

```diff
@@ -4,10 +4,6 @@ import {
   SearchEngine,
   Suggestion,
 } from '@coveo/headless';
-import {
-  QuerySetSection,
-  QuerySuggestionSection,
-} from '@coveo/headless/dist/definitions/state/state-sections';
```

Comment:
This shows how you cant import from anything other than the "exports" defined paths in the package json. Only from @coveo/headless , @coveo/headless/commerce, and so on...